css/layout.css

```diff
@@ -50,24 +69,82 @@
   .column-half {
     width: 50%;
   }
+
+  .navbar {
+    width: 100vw;
+  }
+
+  .search-bar {
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+    margin-right: 5em;
+  }
 }
 
 @media screen and (max-width: 768px) {
   header {
     position: fixed;
   }
 
+  main {
+    padding-top: 4rem;
+  }
+
+  .navbar {
+    width: 100vw;
+    justify-content: space-between;
+  }
+
   .navigation-menu {
     position: fixed;
-    inset: 4.5% 0 70% 0;
+    z-index: -1;
+    inset: 4.5% 0 65% 0;
     background-color: rgb(167, 163, 163, 0.85);
     backdrop-filter: blur(1em);
     flex-direction: column;
     align-items: start !important;
     padding: max(2vh, 1em) 3em;
+    transform: translateY(-120%);
+    transition: transform 350ms ease-out;
+  }
+
+  .navigation-menu[data-visible="true"] {
+    transform: translateY(-2%);
+  }
+
+  .recipe-hr {
+    display: none;
+  }
+
+  .search-bar {
+    display: flex;
+    align-items: center;
+    margin-right: 5em;
+    margin-bottom: 4px;
```

Comment:
Use `rem`, not `px`, for margin sizing: https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css